<launch>
  <node pkg="srproject" name="Object_Recognizition" type="ObjectRecognizition.py" output="screen">

    <remap from="input_rgb_image" to="/camera/rgb/image_color" />
    
    <rosparam>
       LBP_scaleFactor: 1.3
       LBP_minNeighbors: 3
       LBP_minSize: 30
       LBP_maxSize: 150
    </rosparam>

    <param name="template1" value="$(find srproject)/data/templates/Objects/mona_live/Object1.png" />
    <param name="template2" value="$(find srproject)/data/templates/Objects/mona_live/Object3.png" />

   	<param name="cascade_1" value="$(find rbx1_vision)/data/LBP_detectors/cylindercascade.xml" />
  	<param name="cascade_2" value="$(find rbx1_vision)/data/LBP_detectors/conescascade.xml" />

  </node>
</launch>